ctwrap/modules/adiabatic_flame.py

```diff
@@ -94,27 +88,33 @@ def run(name,
     if loglevel > 0:
         f.show_solution()
     msg = '    {0:s}: mixture-averaged flamespeed = {1:7f} m/s'
-    print(msg.format(name, f.u[0]))
+    print(msg.format(name, f.velocity[0]))
 
-    out[name + '<mix>'] = pd.DataFrame(
-        np.vstack([f.grid, f.u, f.V, f.T, f.density, f.X]).T, columns=keys)
+    if name is not 'defaults':
```

Comment:
Do we still have issues with `NaturalNameWarnings`? I'm still somewhat conflicted, but keeping the `.` would create clarity.